.setup/bin/sample_courses/models/__init__.py

```diff
@@ -3,7 +3,8 @@
 """
 from pathlib import Path
 import random
-from sample_courses import *
+import os
+from sample_courses import SETUP_DATA_PATH, DB_ONLY, yaml
```

Comment:
You could consider splitting this into multiple lines.  That way, the git diff is easier to read if someone adds something new.

Something like this perhaps:
```suggestion
from sample_courses import (
    SETUP_DATA_PATH,
    DB_ONLY,
    yaml
)
```